src/components/Adrenaline.jsx

```diff
@@ -8,8 +8,10 @@ const adaptorShape = createAdaptorShape(PropTypes);
 
 export default class Adrenaline extends Component {
   static childContextTypes = {
-    renderLoading: PropTypes.func.isRequired,
-    adaptor: adaptorShape.isRequired,
+    adrenaline: PropTypes.shape({
```

Comment:
I've moved a lot of stuff to context. I think this is stabilizing in React > 0.14 and should be pretty safe to use.